### PR TITLE
bump govmomi to v0.26.0

### PR DESCRIPTION
**Pull request by yastij**
Signed-off-by: Yassine TIJANI <ytijani@vmware.com>

#### What type of PR is this?

/kind cleanup

#### Special notes for your reviewer:

/assign @dims

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Comment by dims** [comment]
@yastij hmmm we just got rid of `go-openapi/spec` i think. will have to look deeper

**Comment by dims** [comment]
@yastij we definitely need to cleanup the `go-openapi/spec`, see the CI job failures please

**Comment by yastij** [comment]
cc @dougm - when did we start depending on `go-openapi/spec` in govmomi ?

**Comment by dougm** [comment]
Hi @yastij , govmomi does not depend on go-openapi , we're not using it anywhere.

File: go.mod [hunk 1]
```diff
@@ -42,6 +42,7 @@ require (
 	github.com/evanphx/json-patch v4.11.0+incompatible
 	github.com/fsnotify/fsnotify v1.4.9
 	github.com/go-bindata/go-bindata v3.1.1+incompatible
+	github.com/go-openapi/spec v0.20.3
```
Comment (liggitt) [inline go.mod:45]: we don't want to add this back in... is this really required by the new govmomi version?

**Comment by liggitt** [comment]
increases deps:

```
2c2
< 	"totalDependencies": 364,
---
> 	"totalDependencies": 368,
4c4
< 	"transitiveDependencies": 238
---
> 	"transitiveDependencies": 241
```

are the additional deps required? can we ask upstream to eliminate them? is this update fixing a critical bug or security issue?

File: go.mod (hunk 1 above)
Comment (yastij) [reply go.mod:45]: agreed. will take a look today to see from where this is coming and see If we can eliminate it upstream

**Pull request closed without merging**